docs/configuration.md

```
@@ -1662,9 +1662,42 @@ fields:
+  [ use_summary_header: <boolean> | default = true ]
+
+  # Emoji name (without colons) to react with on the parent thread message
+  # when all alerts in the group are resolved (e.g. "white_check_mark").
+  # Requires the bot token to have reactions:write scope.
+  [ resolve_emoji: <string> ]
+
+  # Options that only apply when use_summary_header is true (default).
+  [ summary_header: <summary_header_thread_config> ]
+```
+
+##### `<summary_header_thread_config>`
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)